Backend/README.md

```
@@ -0,0 +1,113 @@
+| POST   | `/api/images/upload-image`    | Upload image             |
+| GET    | `/api/images/get-images`      | Retrieve uploaded images |
+
+### 🔐 Auth Route
+
+| Method | Endpoint      | Description                        |
+|--------|---------------|------------------------------------|
+| POST   | `/auth/user`  | Register or validate Google user   |
+
+
+## Tech Stack
+| Node.js / Express.js | Backend API framework   |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)